crates/matrix-sdk-crypto/src/gossiping/machine.rs

```diff
@@ -762,11 +762,6 @@ impl GossipMachine {
             );
             info.sent_out = true;
             self.save_outgoing_key_info(info).await?;
-        } else {
-            warn!(
-                request_id = id.to_string(),
-                "Unknown outgoing secret request; cannot mark as sent"
-            )
```

Comment:
This is totally normal. `GossipMachine::mark_outgoing_request_as_sent` is called for every successfully-sent to-device message, whether or not that request came from the GossipMachine.